dev/scripts/watch-metrics.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# This script is used as a convenience method for monitoring & capturing
+# the HTTP JSON sink for Alluxio metrics
+
+# Alluxio-related arguments
+host=""
+port=""
+metrics_file_prefix=""
+metrics_dir="/tmp/alluxio_metrics"
+
+# Misc. arguments
+time_step=2  # 2s is the default interval for `watch`
+watch=""
+jq_filter=""
+
+USAGE="Usage:
```

Comment:
could you add a couple of sentences describing what this tool will do at a high level? like why should i use it instead of just plain curl?